Base/Python/slicer/util.py

```diff
@@ -928,64 +927,131 @@ def saveScene(filename, properties={}):
 #
 
 def moduleSelector():
+  """Return module selector widget.
+  Throws a RuntimeError exception if there is no module selector (for example, the application runs without a main window).
```

Comment:
There is a way to document exception. See https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html#the-sphinx-docstring-format